.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Fixes #
+
+## Proposed Changes
+
+  -
```

Comment:
@rajitha1998 Shall we add a `Screenshots` section to the PULL_REQUEST_TEMPLATE?

Reply:
sure

Reply:
@janethavi  added screenshots section